plugins/woocommerce/src/Admin/Features/OnboardingTasks/TaskListSection.php

```diff
@@ -0,0 +1,122 @@
+<?php
```

Comment:
This PR is already rather large, but might be good to get some tests around this class in a follow-up.